website/pages/docs/plugins/destinations/bigquery/_meta.json

```diff
@@ -0,0 +1,3 @@
+{
+  "overview": "Overview"
```

Comment:
Why not also a configuration section and configuration.md?

Reply:
I think we can do that in a follow-up, but right now none of the destination plugins have a configuration.md, so let's at least keep it consistent.